model-optimizer/extensions/front/mxnet/arange_like_replacer.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from extensions.ops.elementwise import Add
+from extensions.ops.gather import Gather
+from extensions.ops.range import Range
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementOp
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph
+from mo.ops.shape import Shape
+from mo.ops.reshape import Reshape
+from mo.ops.squeeze import Squeeze
+from mo.utils.error import Error
+
+
+class ArangeLikeReplacer(FrontReplacementOp):
```

Comment:
Please, add description what operation do

Reply:
Added description to mo/ops/arange_like.py